### PR TITLE
Use normal mechanism to install `dotnet-format`

### PR DESCRIPTION
**Pull request by dougbu**
- aspnet/AspNetCore-ManualTests#407
- update the `dotnet-serve` version while here

### CODE REVIEW

**Review by wtgodbe — APPROVED** [review]

**Review by Pilchie — APPROVED** [review]
Thanks - this has bitten me a couple of times when I just run `./build.sh` / `./build.ps1` from a normal shell.

**Review by captainsafia — APPROVED** [review]

File: .config/dotnet-tools.json [hunk 1]
```diff
@@ -3,10 +3,16 @@
...
-      "version": "1.7.125",
+      "version": "1.8.15",
       "commands": [
         "dotnet-serve"
       ]
+    },
+    "dotnet-format": {
+      "version": "5.0.210401",
+      "commands": [
+        "dotnet-format"
```
Comment (pranavkm) [inline .config/dotnet-tools.json:14]: Would we add some spice to our commit hooks by changing this string to `format`?

**Review by pranavkm — APPROVED** [review]

File: .config/dotnet-tools.json (hunk 1 above)
Comment (dougbu) [reply .config/dotnet-tools.json:14]: Please clarify your question @pranavkm

**Comment by dougbu** [comment]
@captainsafia are failures of the following tests a known issue:grey_question:
- Templates.Test.BlazorWasmTemplateTest.BlazorWasmStandaloneTemplate_Works
- Templates.Test.BlazorWasmTemplateTest.BlazorWasmStandalonePwaTemplate_Works

These are failing at least in the f[irst pipeline run for this commit](https://dev.azure.com/dnceng/public/_build/results?buildId=979497).

**Comment by pranavkm** [comment]
Those two tests are the only ones that use `dotnet-serve`. Maybe the update broke it in some way.

**Comment by dougbu** [comment]
:ok: it appears Nate has change the package recently and it's likely the last release (intended to provide `net5.0` support) busted things. /cc @Pilchie for awareness

I'll update this PR to use the previous release of `dotnet-serve`. That (v1.7.139) seems to include only straightforward bugs fixes and dependency updates compared to v1.7.125.

**Pull request merged**